packages/shared/styles/components/SfChevron.scss

```
@@ -0,0 +1,85 @@
+@import '../variables';
+
+$chevron-duration: .5s !default;
+$chevron-easing: cubic-bezier(.25, 1.7, .35, .8) !default;
+$chevron-size-desktop: var(--chevron-size) !default;
+$chevron__bar-width: calc(#{$chevron-size-desktop} / 2) !default;
+$chevron__bar-height: calc(#{$chevron__bar-width} / 4) !default;
+
+$chevron__bar-top-desktop: calc(calc(#{$chevron-size-desktop} / 2) - calc(#{$chevron__bar-height} / 2)) !default;
+$chevron__bar-left-right-desktop: calc(#{$chevron-size-desktop} / 9) !default;
+$chevron__bar-rotate: 45deg !default;
+$chevron__bar-color: var(--chevron-color);
```

Imo you should preset with some color, then you can use this color or replace with the CSS var (when browser has support and respective var is defined), wdyt?

```suggestion
$chevron__bar-color: $c-primary !default;
```